examples/docker-compose/README.md

```diff
@@ -0,0 +1,60 @@
+# Docker Compose + Fluvio
+
+You can run Fluvio Clusters in a Docker Compose setup, this could be useful for
+local development and POC development.
+
+In order to run a Fluvio Cluster through Docker, you will need to run Fluvio
+components separately, we can use Docker Compose `service`s to achieve this.
+
+## Services
+
+- `sc: Streaming Controller`
+- `sc-setup: Post-Initialization Commands`
+- `spu: Streaming Processing Unit`
+
+> To learn more about Fluvio architecture, please refer to [Fluvio Documentation][1]
+
+## Running Locally
+
+Clone this repo using `git clone https://github.com/infinyon/fluvio.git` and
+cd into `./fluvio/examples/docker-compose`, then run `docker compose up --build`.
```

Comment:
I think the "--build" is not needed?

Reply:
Agree!

Reply:
Here is the change: [68d278d](https://github.com/infinyon/fluvio/pull/3910/commits/68d278dfdd847d767aee6070e7eb3e7b9c1eb476)